doc/overview/roadmap.rst

```diff
@@ -196,3 +108,110 @@ recordings, etc.). Some software tasks that shall be completed:
 - Help package OpenMEEG for Debian/Ubuntu
 - Help manage `the continuous integration system
   <https://ci.inria.fr/>`__
+
+
+In progress
+-----------
+
+Diversity, Equity, and Inclusion
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+MNE-Python is committed to recruiting and retaining a diverse pool of
+contributors, see :gh:`8221`.
+
+.. _bids-integration:
+
+BIDS Integration
+^^^^^^^^^^^^^^^^
+MNE-Python is in the process of providing automated analysis of BIDS-compliant
+datasets, see `MNE-BIDS-Pipeline`_. Two key goals are:
+
+1. Incorporating functionality from the
+   `mnefun <https://labsn.github.io/mnefun/overview.html>`__ pipeline,
+   which has been used extensively for pediatric data analysis at `I-LABS`_.
+   Multiple processing steps (e.g., eSSS), sanity checks (e.g., cHPI quality),
+   and reporting (e.g., SSP joint plots, SNR plots) should be ported over.
+2. Adding support for cloud computing. Currently, cloud computing with M/EEG
+   data requires multiple manual steps, including remote environment setup,
+   data transfer, monitoring of remote jobs, and retrieval of output
+   data/results. With the implementation in `MNE-Docker`_, would be achieved
+   in practice by:
+
+  - One-time (or per-project) setup steps, setting up host keys, access tokens,
+    etc.
+  - In code, switch to cloud computing rather than local computing via a simple
+    change of config parameters.
+
+   See also :gh:`6086`.
+
+Statistics efficiency
+^^^^^^^^^^^^^^^^^^^^^
+A key technique in functional neuroimaging analysis is clustering brain
+activity in adjacent regions prior to statistical analysis. An important
+clustering algorithm — threshold-free cluster enhancement (TFCE) — currently
+relies on computationally expensive permutations for hypothesis testing.
+A faster, probabilistic version of TFCE (pTFCE) is available, and we are in the
+process of implementing this new algorithm.
+
+3D visualization
+^^^^^^^^^^^^^^^^
+Historically we have used Mayavi for 3D visualization, but have faced
+limitations and challenges with it. We should work to use some other backend
+(e.g., PyVista) to get major improvements, such as:
+
+1. *Proper notebook support (through vtkjs)* (complete)
```

Comment:
I think it would be more accurate to say:

```suggestion
1. *Proper notebook support (through ipyvtklink)* (complete)
```

A few versions ago, `panel` was the backend used for notebook support but we switched to `ipyvtk` since then (which is based on `ipycanvas`). Sorry for being late to notice this one :pray: